src/devices/bituo_technik.ts

```diff
@@ -18,10 +18,12 @@ const definitions: Definition[] = [
         configure: async (device, coordinatorEndpoint) => {
             const endpoint = device.getEndpoint(1);
             await reporting.bind(endpoint, coordinatorEndpoint, ['haElectricalMeasurement', 'seMetering']);
-            await reporting.currentSummDelivered(endpoint);
-            await reporting.currentSummReceived(endpoint);
+            await reporting.readMeteringMultiplierDivisor(endpoint);
+            await reporting.currentSummDelivered(endpoint, {min: 5, max: 3600, change: 0});
```

Comment:
Only add `change: 0` here and comment the reason why you've added it:

> And I found that our device will only report data when energy and produced energy change by 0.1. Therefore, when the device only measures energy, produced energy will still not be reported, so I changed 'change' to 0, so that both energy and produced can be reported.